src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_FlowAnalysis.cs

```diff
@@ -958,48 +957,169 @@ static void F(string x, string? y)
         [Fact]
         public void NullCoalescingOperator_05()
         {
+            var source0 =
+@"public class A { }
+public class B { }
+public class UnknownNull
+{
+    public A A;
+    public B B;
+}";
+            var comp0 = CreateStandardCompilation(source0, parseOptions: TestOptions.Regular7);
+            comp0.VerifyDiagnostics();
+            var ref0 = comp0.EmitToImageReference();
+
+            var source1 =
+@"public class MaybeNull
+{
+    public A? A;
+    public B? B;
+}
+public class NotNull
+{
+    public A A = new A();
+    public B B = new B();
+}";
+            var comp1 = CreateStandardCompilation(source1, references: new[] { ref0 }, parseOptions: TestOptions.Regular8);
+            comp1.VerifyDiagnostics();
+            var ref1 = comp1.EmitToImageReference();
+
             var source =
 @"class C
 {
-    static void F(C x, Unknown? y)
+    static void F1(UnknownNull x1, UnknownNull y1)
     {
-        (x ?? y).ToString();
-        (y ?? x).ToString();
-        (null ?? y).ToString();
-        (y ?? null).ToString();
+        (x1.A ?? y1.B)/*T:*/.ToString();
     }
-    static void G(C? x, Unknown y)
+    static void F2(UnknownNull x2, MaybeNull y2)
     {
-        (x ?? y).ToString();
-        (y ?? x).ToString();
-        (null ?? y).ToString();
-        (y ?? null).ToString();
+        (x2.A ?? y2.B)/*T:?*/.ToString();
+    }
+    static void F3(MaybeNull x3, UnknownNull y3)
+    {
+        (x3.A ?? y3.B)/*T:*/.ToString();
```

Comment:
>/*T:*/ [](start = 22, length = 6)

What does this mean?

Reply:
The type is an error type which has no string representation.

Reply:
> The type is an error type which has no string representation.

Can we use display format that prints something for error types?

---
In reply to: [175849383](https://github.com/dotnet/roslyn/pull/25358#discussion_r175849383) [](ancestors = 175849383)

Reply:
We could although that might require a new `SymbolDisplay` option since I believe the current choice is no string representation or `?` for error types.